packages/core/src/root.ts

```diff
@@ -117,8 +117,22 @@ export function createRemoteRoot<
       }
 
       if (initialChildren) {
-        for (const child of initialChildren) {
-          normalizedInitialChildren.push(normalizeChild(child, remoteRoot));
+        if (Array.isArray(initialChildren)) {
```

Comment:
With this check, this case will break 

```
    root.createComponent(
      'Button',
      {
        onPress() {
          console.log('Pressed!');
        },
      },
      ['Press me!'],
      'Press me again!'
    ),
  ]),
```

I think it's better to just normalize both `initialChildren` and `moreChildren` by flattening them. Ex:

```
['foo'], 'bar' => ['foo', 'bar']
'foo', 'bar' => ['foo', 'bar']
'foo', ['bar'] => ['foo', 'bar']
'foo', ['bar'], 'xyz' => ['foo', 'bar', 'xyz']
```

Reply:
I intentionally wanted to not support this, and the types should not prevent it. If they have an array, that's all we consider (mostly to prevent this being a breaking change); if they have an array and anything else, they can use the spread operator to turn the array into positional arguments.